chains/solana/contracts/tests/ccip_errors.go

```diff
@@ -0,0 +1,63 @@
+package contracts
```

Comment:
What do you think of grouping the tests into three folders: ccip, mcm, timelock?

Reply:
organized into `ccip` and `mcms` since some timelock/mcms tests have shared dependencies